### ISSUE TITLE
helm keycloak-postgresql resources

### ISSUE THREAD

**Issue by djnews24** [microcks/microcks#1421]
### Reason/Context

Currently no resources are defined for postgres in dem microcks helm chart. 

### Description

It would be good to have default helm config for resources (cpu and mem), runAsUser and runAsGroup and its changeable via values.yaml for the keycloak-postgres.

In some cases thos configs are needed

**Comment by djnews24** [comment]
Hey maybe something like:

resources:
   requests:
       cpu: 100m
       memory: 128Mi
   limits:
       cpu: 150m
       memory: 192Mi

**Comment by lbroudoux** [comment]
Hello @djnews24 
I'm not an expert on PostgreSQL resources but I was wondering if those defaults are not a bit too aggressive. I would not like to degrade the user-experience in favor of little resource savings. Can you confirm these values are enough to provide a good experience?

**Comment by djnews24** [comment]
Hey @lbroudoux ,

i dont know what the best default value should be, but this must be configurable. So the user-experience should not be a problem, or?

**Comment by lbroudoux** [comment]
Yes, you're right, but the defaults should fit all beginners without compromising the experience. I have to make a new deployment on a Kube cluster tomorrow. I'll gather some metrics and check what could be reasonable values.

**Comment by lbroudoux** [comment]
Now available in branch `1.11.x` chart. Waiting the release ;-)
